src/fragments/start/getting-started/react/setup.mdx

```diff
@@ -46,55 +44,35 @@ Project information
+? Select the authentication method you want to use: AWS profile
 
-Distribution directory path (build)
+...
 
-Build command (npm run-script build)
-
-Start command (npm run-script start)
-
-Select the authentication method you want to use: (Use arrow keys)
-❯ AWS profile
-  AWS access keys
-
-# This is the profile you created with the `amplify configure` command in the introduction step.
-Please choose the profile you want to use (Use arrow keys)
+? Please choose the profile you want to use default
 ```
 
-> Where possible the CLI will infer the proper configuration based on the type of project Amplify is being initialized in. In this case it knew we are using Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
+> Amplify CLI will infer the proper configuration based on the type of project Amplify is being initialized in where possible. Here it detected Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
 When you initialize a new Amplify project, a few things happen:
 
-- It creates a top level directory called `amplify` that stores your backend definition. During the tutorial you'll add capabilities such as a GraphQL API and authentication. As you add features, the `amplify` folder will grow with infrastructure-as-code templates that define your backend stack. Infrastructure-as-code is a best practice way to create a replicable backend stack.
-- It creates a file called `aws-exports.js` in the `src` directory that holds all the configuration for the services you create with Amplify. This is how the Amplify client is able to get the necessary information about your backend services.
-- It modifies the `.gitignore` file, adding some generated files to the ignore list
+- It creates a top level directory called **amplify** that stores your backend definition. During the tutorial you'll add capabilities such as a GraphQL API and authentication. As you add features, the **amplify** folder will grow with infrastructure-as-code templates that define your backend stack. Infrastructure-as-code is a best practice way to create a replicable backend stack.
+- It creates a file called **aws-exports.js** in the **src** directory that holds all the configuration for the services you create with Amplify. This is how the Amplify client is able to get the necessary information about your backend services.
+- It modifies the **.gitignore** file, adding some generated files to the ignore list
 - A cloud project is created for you in the AWS Amplify Console that can be accessed by running `amplify console`. The Console provides a list of backend environments, deep links to provisioned resources per Amplify category, status of recent deployments, and instructions on how to promote, clone, pull, and delete backend resources
 
-## Install Amplify libraries
+## Install Amplify
```

Comment:
is there a reason we removed libraries from this title?

Reply:
Install Amplify libraries. Amplify itself is overloaded

Reply:
Should be Amplify Libraries